### PR TITLE
[#EP-938] Stop / Restart gift step 0

### PR DESCRIPTION
**Pull request by Omicron7**
This is the initial Step 0 for Stopping or Restarting a gift modal.

### CODE REVIEW

File: src/app/profile/yourGiving/stopStartRecurringGifts/step0/stopStartStep0.tpl.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<div class="modal-content give-modal-content">
```
Comment (OzzieOrca) [inline src/app/profile/yourGiving/stopStartRecurringGifts/step0/stopStartStep0.tpl.html]: I think https://github.com/CruGlobal/give-web/blob/master/src/common/templates/giveModalWindow.tpl.html will provide this div. `<div class="modal-content give-modal-content" uib-modal-transclude></div>`

File: src/app/profile/yourGiving/stopStartRecurringGifts/step0/stopStartStep0.tpl.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+              <span aria-hidden="true">×</span></button>
+            <h3 translate>What would you like to do?</h3>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-body">
+    <div class="container-fluid" tabindex="-1" role="dialog">
```
Comment (OzzieOrca) [inline src/app/profile/yourGiving/stopStartRecurringGifts/step0/stopStartStep0.tpl.html]: Do we need `tabindex="-1" role="dialog"`? I've been removing them. I think `$uibModal` sets these on a div that is higher up the DOM. Idk why the UX guys have it only on the body. I could be wrong...

Comment (Omicron7) [inline src/app/profile/yourGiving/stopStartRecurringGifts/step0/stopStartStep0.tpl.html]: I think UX team is pulling from core bootstrap for modal markup, which includes those. $uibModal does things slightly different, so you are correct, we don't need em.

File: src/app/profile/yourGiving/stopStartRecurringGifts/step0/stopStartStep0.tpl.html (hunk 1 above)
Comment (Omicron7) [inline src/app/profile/yourGiving/stopStartRecurringGifts/step0/stopStartStep0.tpl.html]: You're correct, this was a copy/paste error.

**Pull request merged**